packages/framework/assets/js/admin/validation/form/validationMailTemplate.js

```diff
@@ -9,7 +9,7 @@ export default function validationMailTemplate () {
         $(this).jsFormValidator({
             'groups': function () {
 
-                const groups = [constant('\\Shopsys\\FrameworkBundle\\Form\\ValidationGroup::VALIDATION_GROUP_DEFAULT')];
+                const groups = [constant('\\Symfony\\Component\\Validator\\Constraint::DEFAULT_GROUP')];
```

Comment:
Is this necessary?

Reply:
Yes, because `\\Shopsys\\FrameworkBundle\\Form\\ValidationGroup::VALIDATION_GROUP_DEFAULT` key does not exist.

Reply:
6f76fd4e04321393b2e2e83c2571da4c6f314642